sdk_container/src/third_party/coreos-overlay/sec-policy/selinux-base/selinux-base-2.20221101-r4.ebuild

```diff
@@ -113,12 +123,22 @@ src_compile() {
 	for i in ${POLICY_TYPES}; do
 		cd "${S}/${i}" || die
 		emake base
+		emake base BINDIR="${ROOT}/usr/bin" NAME=$i SHAREDIR="${ROOT%/}"/usr/share/selinux \
+			LD_LIBRARY_PATH="${ROOT}/usr/lib64:${LD_LIBRARY_PATH}" -C "${S}"/${i}
 		if use doc; then
 			emake html
 		fi
 	done
 }
 
+# ugly but somehow necessary: these files does not seem detected
+# by portage -> they are orphans. When portage remove the previous package
+# these files remain and prevent the symlink to be done after -> we keep the old stuff
+# and the symlinks are not correctly done (they are done on backup folders preventing collision)
+pkg_preinst() {
+	rm --recursive ${ROOT}/etc/selinux/{mcs,mls,targeted}
+}
+
```

Comment:
Or maybe this is something for upstream too?

Reply:
Maybe not related, but I remember getting weird issues when I was testing some packages with `ebuild`. Basically I was doing `ebuild <path to ebuild> install`, check the temporary installation directory, fix something in the ebuild, removed the ebuild installed flag file (`rm -f <whatever weird path>/.installed`) and did `ebuild <path to ebuild> install` again. I actually needed to do `ebuild <path to ebuild> clean` every time instead of removing the flag file.

Anyway, water under the bridge, things seem to be working for you now.